__tests__/pet.test.js/pet.test.js

```diff
@@ -0,0 +1,221 @@
+const Pet = require('../../src/pet');
+
+describe('constructor', () => {
+    it("returns an object", () => {
+        expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+
+
+});
+
+describe('constructor', () => {
+    it('sets the name property', () => {
+        const pet = new Pet('Fido');
+
+        expect(pet.name).toEqual('Fido');
+    });
+});
+
+describe('constructor', () => {
+   it('has a initial age of 0', () => {
+    const pet = new Pet('Fido');
+    
+    expect(pet.age).toEqual(0);
+   });
+    
+});
```

Comment:
Id say since these are all tests related to the constructor they could all be grouped together under the same describe block:

```
describe('creating a pet', () => {
    it("returns an object", () => {
        expect(new Pet('Fido')).toBeInstanceOf(Object);
    });

    it('sets the name property', () => {
        const pet = new Pet('Fido');
    
        expect(pet.name).toEqual('Fido');
    });

    it('has a initial age of 0', () => {
     const pet = new Pet('Fido');
     
     expect(pet.age).toEqual(0);
    });
});
```

As youve named these describe blocks identically the oputput in the terminal when testing isnt any different, but it makes the code tidier. Its also common convention to add a clear line between each of your it blocks


Also, could tests be added to check the initial values for hunger and fitness?